### DISCUSSION TITLE
Cite Voikko in scientific research paper (if any)

### DISCUSSION THREAD

**Discussion by mrgransky** [voikko/discussion#22] [Q&A]
As part of my research work, I used Voikko for spell checking of Finnish documents.
So I would like to know if there might be any research paper for Voikko which I can cite and refer to it in my own research paper reference list.

Cheers,
Farid

**Comment by ylivuoto** [comment]
Terve!

As I see, it seems to be that the literature around Voikko, seems to be very scacre. Some bachelor's level work and some master's level works appears, but nothing more with first the glance.

Bachelor work: https://jyx.jyu.fi/bitstream/handle/123456789/46150/URN%3aNBN%3afi%3ajyu-201506012141.pdf?sequence=1&isAllowed=y
Pro gradu I found: https://trepo.tuni.fi/bitstream/handle/10024/125064/LipsanenJukka.pdf
Pro gradu: https://www.utupub.fi/bitstream/handle/10024/147832/Martikainen_Ville_opinnayte.pdf?sequence=1

These have been written in Finnish (how could it be otherwise) :D Hope this helps a bit!

Cheers,
O-P

**Reply by hatapitk** [reply]
Hi!

Sorry for late reply. There are no research papers on Voikko (that I know of) as Voikko is not a research project, it is a free software project by volunteers outside academic circles. However methods I studied in my master's thesis
  http://urn.fi/URN:NBN:fi:jyu-200809045710

were integrated in Voikko some years after the thesis was written. Thus you could say that this thesis covers the core algorithms behind current version of Voikko even if Voikko is not mentioned in the text.

Harri
